sample/MozillaAllHands16/event

```diff
@@ -1,90 +1,90 @@
-{ 
-	"id": 3,
-	"name": "All Hands/2016 Hawaii",
-	"latitude": 19.924029,
-	"longitude": -155.887498,
-	"location_name": "Hilton Waikoloa village, Hawaii",
-	"start_time": "2016-12-05T13:00:00",
-	"end_time": "2016-12-09T20:00:00",
-	"timezone": "Hawaii-Aleutian",
-	"description": "Multiple team meetings, happening in the same city, at the same time + some opportunity to get together as one big group as well as with other teams as it makes sense. Then, on the last day, we have a fun social event for all, Mozilla-style!",
-	"background_image": "/images/background.jpg",
-	"logo": "/images/mozilla.png",
-	"organizer_name": "Mozilla",
-	"organizer_description": "Mozilla, the proudly non-profit champions of the Internet, helping to keep it healthy, open and accessible to all.",
-	"social_links": [{
-			"id": 1,
-			"link": "https://twitter.com/mozilla",
-			"name": "Twitter"
+{
+	"social-links": [{
+					"id": 1,
+					"link": "https://twitter.com/mozilla",
+					"name": "Twitter"
 		},
 		{
-			"id": 2,
-			"link": "https://github.com/mozilla/mofo-allhands/tree/gh-pages/2016-hawaii",
-			"name": "Github"
+					"id": 2,
+					"link": "https://github.com/mozilla/mofo-allhands/tree/gh-pages/2016-hawaii",
+					"name": "Github"
```

Comment:
@ParthS007 This indentation is incorrect.

Reply:
Thanks @agbilotia1998 , will do the changes